vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go

```diff
@@ -90,14 +104,36 @@ func SetProcessLabel(processLabel string) error {
 	return selinux.SetExecLabel(processLabel)
 }
 
+// SetSocketLabel takes a process label and tells the kernel to assign the
+// label to the next socket that gets created
+func SetSocketLabel(processLabel string) error {
```

Comment:
@runcom Is this something we need to use for `/var/run/docker.sock` (and/or use in containerd to set labels on sockets?)